modules/programs/screen.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.screen;
+
+in
+{
+  options = {
+    programs.screen = {
+      enable = mkEnableOption "GNU screen";
+
+      settings = mkOption {
+        type = types.string;
```

Comment:
The `string` type is mainly kept for backwards compatibility and shouldn't be used in new code (it merges through string concatenation which is typically not what you'd want). Instead use `str` (no merging allowed) or `lines` (merges with a newline separator). In this case `lines` is the correct choice.